.helm/ecamp3/templates/api_configmap.yaml

```diff
@@ -22,3 +22,4 @@ data:
   SENTRY_API_DSN: null,
   {{- end }}
   FRONTEND_BASE_URL: {{ include "frontend.url" . | quote }}
+  RECAPTCHA_SECRET: {{ .Values.recaptcha.secret | quote }}
```

Comment:
Als secret gehört das in die api_secrets.yml. Unterschied zur configmap ist, dass es dann z.B in Kubernetes im Dashboard nie ohne weiteres in Plain Text angezeigt wird.
Ansonsten können secrets genau wie configmap-Werte in einem Pod als Environment Variabeln eingesetzt werden, siehe api_deployment.yml

Reply:
thx